packages/build/src/log/messages/core_commands.js

```diff
@@ -1,17 +1,72 @@
 'use strict'
 
-const { log, logArray, logError } = require('../logger')
+const path = require('path')
+
+const { log, logArray, logErrorSubHeader } = require('../logger')
 const { THEME } = require('../theme')
 
-const logExperimentalEsbuildParameter = function (logs, envVariableName) {
-  logError(
-    logs,
-    `
-You've opted in to an experimental bundling mechanism via the "${envVariableName}" environment variable.
-We recommend against using this functionality in production sites since its behavior may change or it may be removed entirely.
-For the latest updates about function bundling, please visit our Community forum: community.netlify.com.
-`,
-  )
+const logBundleErrorObject = ({ logs, object }) => {
+  const { location = {}, text } = object
+  const { column, file, line, lineText } = location
+  const locationMessage = `${file} ${line}:${column}`
+  const logMessage = `> ${lineText}
```

Comment:
We might want to use `figures` for `>`, for consistency:

https://github.com/netlify/build/blob/cb2ec81f7c1ee62178fa0a3243c8127325f9a9ec/packages/build/src/log/logger.js#L3

This is the symbol we've been using so far: `❯` with a fallback on `>` on systems which do not support that character.

Reply:
Done in https://github.com/netlify/build/pull/2308/commits/156e122c163216b9f55f73e175cbf8edc967d8a5.